be/src/olap/rowset/segment_v2/segment_iterator.cpp

```diff
@@ -571,11 +569,13 @@ Status SegmentIterator::_seek_and_peek(rowid_t rowid) {
         RETURN_IF_ERROR(_seek_columns(_seek_schema->column_ids(), rowid));
     }
     size_t num_rows = 1;
```

Comment:
num_rows is useless

Reply:
👌🏻

Reply:
It stands for every time just seek one row.
If it is useless , clang compilation will failed.